pennylane_lightning/core/src/simulators/lightning_kokkos/utils/tests/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.20)
 
-project(${PL_BACKEND}_utils_tests)
+project(lightning_kokkos_utils_tests)
```

Comment:
@maliasadi Sorry about all that, it was me :) I was thinking it would be easier to copy & paste when we integrate L-GPU.

Reply:
No worries! :smile: